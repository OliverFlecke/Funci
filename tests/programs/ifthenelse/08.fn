main :: Int = let x = []
                in if isEmpty x then 0 else 1
