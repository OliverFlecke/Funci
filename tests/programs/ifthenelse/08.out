Number 0 (Unit [])